src/lib/util.ts

```diff
@@ -40,6 +40,7 @@ export type ChainName =
   | "ethereum"
   | "optimism"
   | "polygon"
+  | "arbitrum"
```

Comment:
is it possible to somehow pull these chain names from `evm.proxies`? we should take the opportunity to make evm.proxies just work so that there's no need to update this file when chains are added

Reply:
Yeah, that should be possible.  I'll set that up

Reply:
We'll do that with a follow up PR.

Reply:
Just chatting with @carsonfarmer looks like this is more complicated than we thought.  If you're ok with it, can we tackle this in a bigger PR after a some planning?

Reply:
sounds good! can you guys capture it in an issue so we can come back to it during planning?